Use native integer type in JSON schema and files

shymega opened the pull request:
Instead of using strings for the image sizes, this commit changes the
JSON schema and JSON files to use an native JSON integer type.

Signed-off-by: Dom Rodriguez <shymega@shymega.org.uk>

nift4 commented:
"type" is hexadecimal, is that supported by JSON?
size needs an app change anyway to avoid crashing

shymega commented:
`type` isn't documented as hex, but I suppose we should keep that as a string. I will edit the PR. `size` should still be a native int.

shymega committed:
Use native integer type in JSON schema and files

Instead of using strings for the image sizes, this commit changes the
JSON schema and JSON files to use an native JSON integer type.

Signed-off-by: Dom Rodriguez <shymega@shymega.org.uk>

nift4 commented:
![Screenshot_20230104_120254](https://user-images.githubusercontent.com/36458596/210541561-b8dc94a7-0a93-4bbf-87e4-0922120f13af.png)
